syntax = "proto3";

message AddFriend {
    int32 fromId = 1;
    int32 toId = 2;
    string addition = 3;
}